girder/api/rest.py

```diff
@@ -473,6 +478,9 @@ def ensureTokenScopes(token, scope):
     :type scope: str or list of str
     """
     tokenModel = ModelImporter.model('token')
+    if tokenModel.hasScope(token, TokenScope.USER_AUTH):
```

Comment:
I take it this is for backwards compatibility? `TokenScope.USER_AUTH` is effectively a super-scope, that always provides permission?